docs/core/prerequisites.md

```diff
@@ -0,0 +1,453 @@
+
+- liblttng-ust0
+- libcurl3 (for 14.x and 16.x)
+- libcurl4 (for 18.x)
+- libssl1.0.0
+- libkrb5-3
+- zlib1g
+- libicu52 (for 14.x)
+- libicu55 (for 16.x)
+- libicu57 (for 17.x)
+- libicu60 (for 18.x)
+
+For .NET Core 2.0, following dependencies are also required:
+
+- libunwind8
+- libuuid1
+
+For .NET Core applications that use the *System.Drawing.Common* assembly, you also need the following dependency:
+
+- libgdiplus (version 6.0.1 or later)
+
+> [!WARNING]
+> Most versions of Ubuntu include an earlier version of libgdiplus. You can install a recent version
+> of libgdiplus by adding the Mono repository to your system. For more information,
+> see <https://www.mono-project.com/download/stable/>.
+
+### CentOS and Fedora
+
+CentOS distributions require the following libraries installed:
+
+- lttng-ust
+- libcurl
+- openssl-libs
+- krb5-libs
+- libicu
+- zlib
+
+Fedora users: If your openssl's version >= 1.1, you'll need to install compat-openssl10.
+
+For .NET Core 2.0, following dependencies are also required:
+
+- libunwind
+- libuuid
+
+For more information about the dependencies, see [Self-contained Linux applications](https://github.com/dotnet/core/blob/master/Documentation/self-contained-linux-apps.md).
+
+For .NET Core applications that use the *System.Drawing.Common* assembly, you'll also need the following dependency:
+
+- libgdiplus (version 6.0.1 or later)
+
+> [!WARNING]
+> Most versions of CentOS and Fedora include an earlier version of libgdiplus. You can install a recent version
+> of libgdiplus by adding the Mono repository to your system. For more information,
+> see <https://www.mono-project.com/download/stable/>.
+
+::: zone-end
+
+::: zone pivot="os-macos"
+
+# [.NET Core 3.0](#tab/netcore30)
+
+.NET Core 3.0 is supported on **macOS High Sierra (version 10.13)** and later versions. An **x64** CPU architecture is required.
+
+For more information about .NET Core 3.0 supported OS, distributions, and lifecycle policy, see [.NET Core 3.0 Supported OS Versions](https://github.com/dotnet/core/blob/master/release-notes/3.0/3.0-supported-os.md).
+
+# [.NET Core 2.2](#tab/netcore22)
+
+.NET Core 2.2 is supported on **macOS Sierra (version 10.12)** and later versions. An **x64** CPU architecture is required.
+
+For more information about .NET Core 2.2 supported OS, distributions, and lifecycle policy, see [.NET Core 2.2 Supported OS Versions](https://github.com/dotnet/core/blob/master/release-notes/2.2/2.2-supported-os.md).
+
+# [.NET Core 2.1](#tab/netcore21)
+
+.NET Core 2.1 is supported on **macOS Sierra (version 10.12)** and later versions. An **x64** CPU architecture is required.
+
+For more information about .NET Core 2.1 supported OS, distributions, and lifecycle policy, see [.NET Core 2.1 Supported OS Versions](https://github.com/dotnet/core/blob/master/release-notes/2.1/2.1-supported-os.md).
+
+---
+
+::: zone-end
+
+::: zone pivot="os-windows"
+
+## Install with Visual Studio
```

Comment:
@richlander has mentioned that he'd like an article about set up your dev machine. This content from here on out starts to feel it's not prereqs anymore and should be in a different article.

Reply:
This is true, it's something I've pointed out about the windows prereqs article in the past. However, this issue is just a step in that bigger picture, this issue is just cleaning up those prereqs and merging the experience. I envision more PRs that break out the story appropriately.

Reply:
However... I'll look at it more. I think really we should just drop the "prerequisites" page and thoughts all together. I think we can easily have this just become "install" which lists the dependencies. I'm not even seeing the word prerequisites show that much up in skyray for keywords..

Reply:
Yep but there are prereqs for running Core on VS, so that’s why we had it there.